packages/core/src/graph/controller/view.ts

```diff
@@ -184,6 +184,73 @@ export default class ViewController {
     }
   }
 
+  public fitItems(items: Item[], animate?: boolean, animateCfg?: GraphAnimateConfig): void {
+    if (!items.length) {
+      this.fitView(animate, animateCfg);
+      return;
+    }
+
+    const { graph } = this;
+    const padding = this.getFormatPadding();
+    const width: number = graph.get('width');
+    const height: number = graph.get('height');
+    const group: IGroup = graph.get('group');
+    const startMatrix = group.getMatrix() || [1, 0, 0, 0, 1, 0, 0, 0, 1];
+    group.resetMatrix();
+
+    let bbox: BBox = {
+      x: 0, y: 0,
+      minX: Number.MAX_SAFE_INTEGER, minY: Number.MAX_SAFE_INTEGER,
+      maxX: 0, maxY: 0,
```

Comment:
The values for `maxX` and `maxY` should be initialized to `Number.MIN_SAFE_INTEGER` instead of zero as the bounding box might fit entirely in negative coordinates.